root/index.html

```diff
@@ -31,6 +31,11 @@ <h1>Wodin</h1>
         Infectious Disease Modelling with a focus on Malaria, run in
         The Gambia
       </li>
+      <li>
+        <a href="/gambia-idm-2024"><code>gambia-idm-2024</code></a>:
+        Infectious Disease Modelling with a focus on Malaria, run in
+        The Gambia
```

Comment:
```suggestion
        The Gambia in 2024
```
Could add year here and in the previous item to distinguish the two.